Documentation/contributing/testing/ci.rst

```diff
@@ -153,103 +153,73 @@ packet.net before the branch is merged.
-
-For a full list of Jenkins Jobs, see `Jenkins
+allowed to trigger these jobs.
+
+Depending on the PR target branch, a specific set of jobs is marked as required.
+They will be featured in PR checks directly on the PR page. The following
+trigger phrases may be used to trigger them all at once:
+
++------------------+--------------------------+
+| PR target branch | Trigger required PR jobs |
++==================+==========================+
+| master           | test-me-please           |
++------------------+--------------------------+
+| v1.9             | test-backport-1.9        |
++------------------+--------------------------+
+| v1.8             | test-backport-1.8        |
++------------------+--------------------------+
+| v1.7             | test-backport-1.7        |
++------------------+--------------------------+
+
+On top of these shorthands for triggering required jobs, one may use any
+of these additional triggers to either trigger optional jobs, or manually
+re-trigger a single job that previously failed (in case of a flake):
+
++--------------------+---------------------------------------------------------------------------+
+| Trigger phrase     | PR job(s) triggered                                                       |
++====================+===========================================================================+
+| test-X.XX-4.9      | Specific K8s version ``X.XX`` on Kernel 4.9                               |
++--------------------+---------------------------------------------------------------------------+
+| test-X.XX-4.19     | Specific K8s version ``X.XX`` on Kernel 4.19                              |
++--------------------+---------------------------------------------------------------------------+
+| test-X.XX-net-next | Specific K8s version ``X.XX`` on Kernel net-next                          |
++--------------------+---------------------------------------------------------------------------+
+| test-gke           | GKE test (runs on K8s version currently marked as "stable" in GKE)        |
++--------------------+---------------------------------------------------------------------------+
+| test-runtime       | Runtime test                                                              |
++--------------------+---------------------------------------------------------------------------+
+| test-upstream      | Upstream test (note: K8s and Kernel versions vary depending on PR branch) |
++--------------------+---------------------------------------------------------------------------+
+| test-4.9           | Latest K8s version on Kernel 4.9                                          |
++--------------------+---------------------------------------------------------------------------+
+| test-4.19          | Latest K8s version on Kernel 4.19                                         |
```

Comment:
Is this correct? I thought `test-4.19` triggered Cilium-K8s-1.19-kernel-4.19 on master.

Reply:
It is, but poorly written... Amongst all the 4.19 jobs, it triggers the one with the highest K8s version, but I'm not sure how to convey it better... Any ideas? `Kernel 4.19 job with highest tested K8s version (see matrix)`? 😕

Reply:
Well, technically `Penultimate K8s version on kernel 4.19` but I'm not sure that will be clear to everyone. Otherwise, `Second (to) latest K8s version on kernel 4.19`.

Reply:
I was thinking about it some more last night, and actually believe it'd be better to remove them. We already have triggers for individual jobs, better use them rather than have additional shorthands that move from version to version.

Reply:
I like that they are shorter versions for the jobs we need to retrigger most often, but you're probably right.

Reply:
Removed in latest push.